subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -99,6 +99,18 @@ Miscellaneous::
   Use link:{javadocPath}/org/gradle/process/CommandLineArgumentProvider.html[org.gradle.process.CommandLineArgumentProvider] instead.
 * The bridge method `org.gradle.api.tasks.TaskInputs.property(String name, @Nullable Object value)` returning `TaskInputs` has been removed.
   A plugin using the method must be compiled with Gradle 4.3 to work on Gradle 6.0.
+* The setters for
```

Comment:
I think this could be clearer with regards to what you need to do to remediate.  I had to stop and think about it when I first read this and I already understood the change.  We should probably provide an example:

Use the corresponding getter and call `setFrom` on the returned `ConfigurableFileCollection`.  For example, replace `JacocoReport.setExecutionData()` with `JacocoReport.getExecutionData().setFrom()`.

Alternatively, we could provide a bullet list of removed method/"use this now" items:

- The following setters have been removed on `JacocoReport`:
  - `setExecutionData()` - use `getExecutionData().setFrom()` instead.
  - `setSourceDirectories()` - use `getSourceDirectories().setFrom()` instead.

or something like that.

Reply:
I opted for the bullet list.